src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Routes, Route, Navigate,
+} from 'react-router-dom';
+import Layout from './components/Layout';
+import Boats from './components/pages/Boats';
+import BoatDetails from './components/pages/BoatDetails';
+import MyReservations from './components/pages/MyReservations';
+import Login from './components/pages/Login';
+import Signup from './components/pages/Signup';
+import Reserve from './components/pages/Reserve';
+import DeleteBoat from './components/pages/DeleteBoat';
+import AddBoat from './components/pages/AddBoat';
+
+const App = () => (
+  <Router>
```

Comment:
- [ ] I can't delete boats:

![image](https://github.com/Felipe-Perez-Ferraro/final_capstone_front/assets/57421823/4f9f3b83-9117-4c5c-a81e-9b4eb7610dc5)

This may be related to the last review comment (the server thinks my username is wrong).

Kindly, make sure your users can delete boats so your app works as intended. 👍